#! /bin/sh /usr/share/dpatch/dpatch-run
## 05_term.h.dpatch by Steve McIntyre <steve.mcintyre@linaro.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: ncursesw5 include file path change.

@DPATCH@

--- pinfo-0.6.9.old/src/utils.c	2006-03-16 14:14:30.000000000 +0000
+++ pinfo-0.6.9/src/utils.c	2011-12-13 16:48:17.783029574 +0000
@@ -50,7 +50,7 @@
 #ifdef HAS_READLINE
 #include <readline/readline.h>
 #include <readline/history.h>
-#include <term.h>
+#include <ncursesw/term.h>
 /* HAS_READLINE */
 #endif
 
